src/event_form/steps/event-form-step5.html

```diff
@@ -454,9 +232,52 @@ <h4 class="modal-title">Weergave</h4>
                                ng-change="validateInfo(); saveContactInfo();"
                                ng-model-options="{ updateOn: 'blur' }"/>
                         <span class="help-block" ng-hide="infoErrorMessage === ''" ng-bind="infoErrorMessage"></span>
+
+                          <div class="checkbox">
```

Comment:
Bovenliggende switch statement gebruiken om de verschillende label elementen per type te tonen. in `showBookingOption` zit al een 'type check'.